db/migrate/20141015173220_add_docker_image_fields.rb

```diff
@@ -1,35 +0,0 @@
-class AddDockerImageFields < ActiveRecord::Migration[4.2]
-  def up
-    add_column :docker_images, :katello_uuid, :string, :limit => 255
-    add_column :docker_images, :katello_repository_id, :integer
-    add_column :docker_tags, :katello_repository_id, :integer
-
-    create_table :katello_repository_docker_images do |t|
-      t.references :docker_image, :null => false
-      t.references :repository, :null => true
-    end
-
-    add_index :docker_images, :katello_uuid, :unique => true
-
-    add_index :katello_repository_docker_images, [:docker_image_id, :repository_id],
-              :name => :katello_repo_docker_imgs_image_repo_id, :unique => true
-
-    add_index :docker_tags, [:docker_image_id, :katello_repository_id, :tag],
-              :name => :katello_repo_docker_tags_image_repo_id, :unique => true
```

Comment:
Should this index be dropped?

Reply:
This table doesn't exist anymore and is dropped in future migrations anyway. This change  LGTM